src/features/orientation/components/selected-filters/selected-filters.component.html

```diff
@@ -1,44 +1,53 @@
-<ng-container *ngIf="filterPresentation as filter">
-  <div class="form-check form-check-inline form-chip mb-3" *ngIf="filter.services">
-    <button class="chip-button" (click)="resetForm(filter.services, 'services')">
-      {{ filter.services }}
+<ng-container *ngIf="filterPresentation as filter" style="flex-wrap: wrap" class="d-flex">
```

Comment:
```suggestion
<ng-container *ngIf="filterPresentation as filter">
```
Je ne crois pas que `ng-container` puisse prendre en compte les classes et les styles à tester sans

Si c'est le cas, remplacer `style="flex-wrap: wrap" class="d-flex"` par la classe `flex-wrap` : https://getbootstrap.com/docs/5.2/utilities/flex/#wrap